addons/zeus/functions/fnc_ui_attributeCargo.sqf

```diff
@@ -19,18 +19,75 @@
 params ["_control"];
 TRACE_1("params",_control);
 
-private _veh = GETMVAR(BIS_fnc_initCuratorAttributes_target,objNull);
-TRACE_1("",_veh);
+private _vehicle = GETMVAR(BIS_fnc_initCuratorAttributes_target,objNull);
+TRACE_1("",_vehicle);
 
-private _loaded = _veh getVariable [QEGVAR(cargo,loaded), []];
+private _loaded = _vehicle getVariable [QEGVAR(cargo,loaded), []];
 TRACE_1("",_loaded);
 
-_control ctrlRemoveAllEventHandlers "setFocus";
+_control ctrlRemoveAllEventHandlers "SetFocus";
 
+// Init cargo list
 private _listbox = _control controlsGroupCtrl 80086;
 
 {
     private _class = if (_x isEqualType "") then {_x} else {typeOf _x};
     private _displayName = getText (configFile >> "CfgVehicles" >> _class >> "displayName");
     _listbox lbAdd _displayName;
 } forEach _loaded;
+
+// Init unload button
+private _button = _control controlsGroupCtrl 80087;
+
+private _fnc_onButtonUnload = {
+    params ["_button"];
+
+    // Validate vehicle
+    private _vehicle = GETMVAR(BIS_fnc_initCuratorAttributes_target,objNull);
+    if (isNull _vehicle || {!alive _vehicle}) exitWith {
+        LOG("Vehicle deleted or killed, cannot unload");
```

Comment:
It's a been a while, but is log a valid macro? I ask because it's an sqf command 🤔

Reply:
Yes it is.

Reply:
@SilentSpike Don't remember
`#define true false` fun? ^^
The preprocessor doesn't know what SQF is. It just does it's job and hands the result off to others.

Reply:
😆 Yeah I remembered, that's why I wasn't sure if we'd actually hijacked that command